src/components/ArticleList.jsx

```diff
@@ -0,0 +1,23 @@
+const ArticleList = ({ articles }) => {
+  return (
+    <ul className="article-list">
```

Comment:
It would be great to see a loading pattern here. The user would ideally want to know things are working.